blueocean-config/gulpfile.js

```diff
@@ -1,7 +0,0 @@
-var builder = require('@jenkins-cd/js-builder');
```

Comment:
The READMEs refer to gulp all over... need some work done to replace this. Assumig people won't just be able ot run gulp anymore... directly (as no gulpfile) :ant: